frontend/src/metabase-lib/order_by.ts

```diff
@@ -0,0 +1,21 @@
+import * as ML from "cljs/metabase.lib.js";
+import type { Field } from "metabase-types/api";
+import type { OrderByClause, Query } from "./types";
+
+export function orderableColumns(query: Query): Field[] {
+  return ML.orderable_columns(query);
+}
+
+export function orderBys(query: Query): OrderByClause[] {
+  const result = ML.order_bys(query);
+  return result === null ? [] : result;
+}
```

Comment:
Instead of doing stuff like this I think we can just change `order_bys` to always return something if that's what you would prefer.

In Clojure having it return `nil` instead of `[]` is a little nicer I think, because you can do

```clj
(if (order-bys query)
  ...
  ...)
```

As opposed to 

```clj
(if (seq (order-bys query))
  ...
  ...)
```

but it's not a dramatic difference.

Reply:
@camsaul, that'd be cool if it doesn't make life harder for the BE — I believe on the FE we'd rather expect an empty array in cases like that